kivy/data/style.kv

```diff
@@ -231,7 +231,7 @@
             rgba: 1, 1, 1, int(not self.is_leaf)
         Rectangle:
             source: 'atlas://data/images/defaulttheme/tree_%s' % ('opened' if self.is_open else 'closed')
-            size: 16, 16
+            size: self.height / 2.0, self.height / 2.0
             pos: self.x - 20, self.center_y - 8
```

Comment:
I guess changing `size` only isn't enough if you introduce `self.height / 2.0`.

Reply:
Should be? Size is a ReferenceListProperty for width and height, so changing size would also change these two properties. Do you have reason to believe otherwise?

Reply:
My edit didn't pass O_o ...anyway, I'm talking about `pos` because currently there's just fixed amount subtracted :)